### PR TITLE
Kogito Travel Agency support for Business Key

### PR DESCRIPTION
**Pull request by cristianonicolai**
- Auto generated travel code
- Added missing changes from previous demos: favicon and cors

### CODE REVIEW

File: kogito-travel-agency/travels/src/main/resources/META-INF/resources/index.html [hunk 1]
```diff
@@ -107,11 +108,22 @@ <h1>Travels</h1>
...
 
     function create(travelRequest) {
-        $.post("/travels", travelRequest, function () {
+        var key = makeid();
+        $.post("/travels?businessKey=" + key, travelRequest, function () {
             load();
         }, "json");
     }
 
+    function makeid() {
```
Comment (ricardozanini) [inline kogito-travel-agency/travels/src/main/resources/META-INF/resources/index.html:117]: https://github.com/uuidjs/uuid

:stuck_out_tongue:

**Review by ricardozanini — APPROVED** [review]

File: kogito-travel-agency/travels/src/main/resources/META-INF/resources/index.html (hunk 1 above)
Comment (cristianonicolai) [reply kogito-travel-agency/travels/src/main/resources/META-INF/resources/index.html:117]: I'm actually generating something like a reservation code ( 6 digits upper case ), like the one we get when doing a flight booking :) so not exactly the same thing as uuid

Comment (ricardozanini) [reply kogito-travel-agency/travels/src/main/resources/META-INF/resources/index.html:117]: I'm just annoying you!

**Review by AjayJagan — APPROVED** [review]
cool! <3

**Pull request merged**